lib/peep.rb

```diff
@@ -0,0 +1,50 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :id, :message, :date
+
+  def initialize(id:, message:, date:)
+    @id = id
+    @message = message
+    @date = date
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
You're using this same piece of code in more than one place in this file. How might you get rid of this duplication?